modules/configuring-a-floating-action-button/proc-configuring-floating-action-button-as-a-dynamic-plugin.adoc

```diff
@@ -0,0 +1,158 @@
+:_mod-docs-content-type: PROCEDURE
+[id="proc-configuring-floating-action-button-as-a-dynamic-plugin_{context}"]
+= Configuring a floating action button as a dynamic plugin
+
+You can configure a dynamic plugin as a floating action button to perform actions or open an internal or external link.
+
+.Procedure
+
+. Add the `app-config-dynamic.yaml` content into `app.config-local.yaml`.
+
+. Set the disabled property to `false`. For example:
++
+[source,yaml]
+----
+global:
+  dynamic:
+    includes:
+      - dynamic-plugins.default.yaml
+    plugins:
+      - package: ./dynamic-plugins/dist/backstage-plugin-global-floating-action-button
+        disabled: false
+----
+
+. To configure a dynamic plugin as a floating action button, complete any of the following tasks:
+
+** Specify the `global.floatingactionbutton/config` mount point in your plugin configuration. For example: 
++
+.bulk-import plugin as a floating action button example
+[source,yaml]
+----
+  dynamicPlugins:
+    frontend:
+      red-hat-developer-hub.backstage-plugin-bulk-import:
+        # Start of the floating action button configuration
+        mountPoints:
+          - mountPoint: global.floatingactionbutton/config
+            importName: BulkImportPage <1>
+            config:
+              slot: 'page-end'
+              icon: bulkImportIcon
+              label: 'Bulk import'
+              toolTip: 'Register multiple repositories in bulk'
+              to: /bulk-import/repositories
+        # End of the floating action button configuration
+        appIcons:
+          - name: bulkImportIcon
+            importName: BulkImportIcon
+        dynamicRoutes:
+          - path: /bulk-import/repositories
+            importName: BulkImportPage
+            menuItem:
+              icon: bulkImportIcon
+              text: Bulk import
+----
+<1> (Required) The import name with an associated component to the mount point.
+
+** To configure a dynamic plugin as a floating action button that opens an external link, specify the `global.floatingactionbutton/config` mount point in the `backstage-plugin-global-floating-action-button` plugin. For example:
++
+.floating action button that opens Github example
+[source,yaml]
+----
+  dynamicPlugins:
+    frontend:
+      red-hat-developer-hub.backstage-plugin-global-floating-action-button:
+        mountPoints:
+          - mountPoint: application/listener
+            importName: DynamicGlobalFloatingActionButton
+          - mountPoint: global.floatingactionbutton/config
+            importName: NullComponent
+            config:
+              icon: github
+              label: 'Git'
+              toolTip: 'Github'
+             	to: https://github.com/redhat-developer/rhdh-plugins
+----
+
+.Floating action button parameters
```

Comment:
I'm not sure if this table fits in this procedural module. Maybe a subsection for these parameters. wdyt?
Or maybe this table can be added at the beginning of the section.